test/integration/src/test/java/io/pravega/test/integration/endtoendtest/EndToEndTruncationTest.java

```diff
@@ -352,8 +352,6 @@ public void testSimpleOffsetTruncation() throws ReinitializationRequiredExceptio
         final EventStreamReader<String> newReader = clientFactory.createReader(newReaderGroupName + "2",
                 newReaderGroupName, new JavaSerializer<>(), ReaderConfig.builder().build());
 
-        // Check that we get a TruncatedDataException after truncating the active segment and then read the expected event.
-        assertThrows(TruncatedDataException.class, () -> newReader.readNextEvent(1000).getEvent());
```

Comment:
yes, the test expected offset to be zero following a truncation and hence asserted to receive a `truncatedException`. 
but now that we send the correct offset, in this scenario the client should not receive truncatedException. hence these asserts are removed.

Reply:
@fpj The new reader threw a `TruncatedDataException` since the controller returned offset 0L. The PR ensures that the controller returns the truncated offset and hence `TruncatedDataException` will not be observed.

Reply:
Ok.